test/test_autoencoder.py

```diff
@@ -6,13 +6,45 @@
 from carla.data.catalog import DataCatalog
 from carla.models.catalog import MLModelCatalog
 from carla.recourse_methods.autoencoder import (
+    CSVAE,
     Autoencoder,
     VariationalAutoencoder,
     train_autoencoder,
     train_variational_autoencoder,
 )
 
 
+def test_cs_vae():
+    # Build data and mlmodel
+    data_name = "adult"
+    data = DataCatalog(data_name)
+
+    model = MLModelCatalog(data, "ann", backend="pytorch")
+
+    test_input = np.zeros((1, 20))
+    test_input = torch.Tensor(test_input)
+    test_class = torch.Tensor(np.array([[0, 0]]))
+
+    csvae = CSVAE(data_name, layers=[test_input.shape[1], 16, 8])
+
+    fitted_csvae = train_variational_autoencoder(
+        csvae, data, model.scaler, model.encoder, model.feature_input_order, epochs=1
+    )
```

Comment:
I saw this in CEM too, right now for the testing we train autoencoders which makes the whole test quite slow if we have multiple of the recourse methods where we train these. Would it be ok for all of them to check if there is a trained autoencoder available?

Reply:
I'm not sure if this will work with our CI 😅 On one hand, we're using different types of autoencoders and can't reuse them for other methods, and on the other hand, I don't know how and where the github process saves trained models.

A check for trained models sounds good, I'll add a Issue for that 👍